log/helper.go

```diff
@@ -66,18 +66,29 @@ func (h *Helper) WithContext(ctx context.Context) *Helper {
 	}
 }
 
+func (h *Helper) preCheckLevel(level Level) bool {
```

Comment:
maybe we can use a more accurate method name, like https://github.com/uber-go/zap/blob/8f5ee80ab2dbc713823341ce30334cd9c03a98e5/zapcore/level.go#L213

Reply:
fixed.